public/locales/en/widgets/location.json

```diff
@@ -0,0 +1,34 @@
+{
+  "form": {
+    "field": {
+      "query": "City / postal code",
+      "latitude": "Latitude",
+      "longitude": "Longitude"
+    },
+    "button": {
+      "search": {
+        "label": "Search",
+        "disabledTooltip": "Please choose a city / postal code first"
+      }
+    },
+    "empty": "Unknown location"
+  },
+  "modal": {
+    "title": "Choose a location",
+    "table": {
+      "header": {
+        "city": "City",
+        "country": "Country",
+        "coordinates": "Coordinates",
+        "population": "Population"
+      },
+      "action": {
+        "select": "Select {{city}}, {{countryCode}}"
+      },
+      "population": {
+        "fallback": "Unknown",
+        "count": "{{count}} people"
```

Comment:
I don't think we need the people, this translation feels a bit useless. Also maybe we could humanize it so it like 10k, 12M. But it's not necessary